django_ses/views.py

```diff
@@ -282,10 +305,81 @@ def handle_bounce(request):
                     },
                 )
 
-                signals.delivery_received.send(
-                    sender=handle_bounce,
+                if signal_sender == 'handle_bounce':
+                    signals.delivery_received.send(
+                        sender=handle_bounce,
+                        mail_obj=mail_obj,
+                        delivery_obj=delivery_obj,
+                        raw_message=raw_json,
+                    )
+                else:
+                    signals.delivery_received.send(
+                        sender=handle_event,
+                        mail_obj=mail_obj,
+                        delivery_obj=delivery_obj,
+                        raw_message=raw_json,
+                    )
+            elif event_type == 'Send':
+                # Send
+                send_obj = message.get('send', {})
+
+                # Logging
+                feedback_id = send_obj.get('feedbackId')
+                feedback_type = send_obj.get('deliveryFeedbackType')
+                logger.info(
+                    u'Received send notification: feedbackId: %s, feedbackType: %s',
+                    feedback_id, feedback_type,
+                    extra={
+                        'notification': notification,
```

Comment:
Just confirming, is this serializable at all times? Wouldn't want to throw an error in custom loggers.

Reply:
I guess it is serializable at all times, because `notification` is an output of `json.loads`.